index.bs

```diff
@@ -71,8 +74,32 @@ Examples {#examples}
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Ambient Light Sensor provides information about lighting conditions near
+the device environment. Potential privacy risks include:
+
+  - Information leaks about the user’s surrounding and work habits.
+  - Profiling. Readout from Ambient Light Sensor can potentially induce
+    information leaks about the user’s interests, web use and work
```

Comment:
Hello indeed those were too elaborate. Feel free to simplify, i.e.

"work habits" -> "use patterns"

Profiing - you are right, unnecessary repeat.